os/customizing-docker.md

```diff
@@ -95,13 +93,43 @@ cp -p ~/cfssl/client.pem cert.pem
 cp -p ~/cfssl/client-key.pem key.pem
 ```
 
-### Drop-in Configuration
+### Enable the Secure Remote API on a New Socket
+
+**NOTE:** Skip this part if you use CoreOS releases older than 949.0.0.
```

Comment:
In most circumstances it makes since to use a Github link...in this one it doesn't since the link doesn't exist.